extensions/blocks/opentable/edit.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * External dependencies
+ */
+import 'url-polyfill';
```

Comment:
I think you'll need to specify a source here.

Reply:
I'm not 100% sure what you mean here. This is how it's being [imported elsewhere](https://github.com/Automattic/jetpack/blob/7ee491958b6652baa38bcad791f2a36b151e504d/modules/search/instant-search/lib/query-string.js#L4) not that this means it's the right way to do it of course!

Reply:
I read somewhere that all extensions receive all the polyfills by default, so maybe this line isn't needed at all?

Reply:
> I read somewhere that all extensions receive all the polyfills by default,

It's only the WordPress polyfills though, as far as I know:
https://developer.wordpress.org/block-editor/contributors/develop/scripts/#polyfill-scripts

> This is how it's being imported elsewhere not that this means it's the right way to do it of course!

True, my bad! So I take it I don't need to specify the source like so?

```suggestion
import 'url-polyfill' from 'url-polyfill';
```